accord-core/src/main/java/accord/local/Bootstrap.java

```diff
@@ -273,7 +273,7 @@ public void fail(Ranges ranges, Throwable failure)
 
             store.agent().onFailedBootstrap(attempt, "PartialFetch", newFailures, () -> {
                 node.scheduler().selfRecurring(() -> {
-                    store.execute((PreLoadContext.Empty) () -> "Restart Bootstrap", safeStore -> restart(safeStore, newFailures.slice(allValid, Minimal), attempt + 1), store.agent());
+                    store.execute((PreLoadContext.Empty) () -> "Restart Bootstrap", safeStore -> { restart(safeStore, newFailures.slice(allValid, Minimal), attempt + 1); }, store.agent());
```

Comment:
Noticed we have to disambiguate between Consumer and Function by adding brackets, but at least in Accord the Function overload is never used. Should we keep it?

Reply:
good spot, let me check if we use it in C*. If not I'll remove it, would be nice to clean that up